app/components/ImageBox.tsx

```diff
@@ -17,28 +33,52 @@ function ImageBox({
   className,
   tagName,
   plausibleEventName,
+  border,
+  size,
+  zoomable = true,
+  background = false,
 }: Readonly<ImageBoxProps>) {
   if (!image || !image.url) return null;
   const headingTag = tagName ? tagName : "h3";
   const imageForZoomable: ImageProps = {
     ...image,
     className: twJoin(image.className, "border-[2px] border-[#004B760D]"),
   };
+
+  const isBigImage = size === "MEDIUM" || size === "LARGE";
+
   return (
-    <figure className={twJoin("pb-[40px]", className)}>
+    <figure
+      className={twMerge(className, size && twJoin("w-full", imageSizes[size]))}
+    >
       {title && (
         <Heading className="ds-heading-03-reg pb-[24px]" tagName={headingTag}>
           {title}
         </Heading>
       )}
-      <ImageZoomable
-        image={imageForZoomable}
-        plausibleEventName={plausibleEventName}
-      />
-      <figcaption className="label-03-reg pt-[24px]">
-        {image.caption}
-      </figcaption>
-      <hr className="my-[16px] w-[120px] border-t-[2px] border-blue-300" />
+      {zoomable && isBigImage ? (
+        <ImageZoomable
+          image={imageForZoomable}
+          plausibleEventName={plausibleEventName}
+          className="object-cover"
+          square={!!size}
+        />
+      ) : (
+        <Image {...image} className="object-contain" />
+      )}
+      {image.caption && isBigImage && (
```

Comment:
Same here, why is the decision of a caption implicitly prevented if it's small? Shouldn't we are in charge if we want to show a caption or not no matter of the size? Shouln't that be fixed by design, not by code?